src/components/Animal/Animal.js

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import './Animal.css';
 
-export default function Animal() {
-  return <></>;
+export default function Animal(props) {
+  return (
+    <div className="animal" style={{ left: props.left, top: props.top }} >
+      <img src={`${process.env.PUBLIC_URL}/animals/${props.type}.svg`} />
+      <span className='name'>{props.name}</span>
+      <span>{props.says}</span>
+    </div>
+  );
```

Comment:
Very nice work Mary! Nice job adding the class of 'animal' to the div and 'name' to the span tag and correctly passing props down to the the Animal component.